### PR TITLE
Revise landing pages

### PR DESCRIPTION
**Pull request by amarantha-k**

### CODE REVIEW

File: home/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,93 +1,256 @@
...
+[.column]
+====== {empty}
+.SDK and Connectors
 
 [.content]
-An open source, distributed, JSON document database. It exposes a scale-out, key-value store with managed cache for sub-millisecond data operations, purpose-built indexers for efficient queries, and a powerful query engine for executing SQL-like queries.
+Couchbase SDKs allow applications to access a Couchbase cluster and the big data Connectors enable data exchange with other platforms.
 
-xref:server:getting-started:start-here.adoc[Get Started: Couchbase Server Enterprise]
+xref:sdk.adoc[Developer Docs]
```
Comment (RichardSmedley) [inline home/modules/ROOT/pages/index.adoc:97]: Should be * xref:home:ROOT:sdk.adoc[Developer Docs]]
?

Comment (amarantha-k) [reply home/modules/ROOT/pages/index.adoc:97]: Since sdk.adoc is in the same directory, we don't need to specify the full path.

Comment (amarantha-k) [reply home/modules/ROOT/pages/index.adoc]: updated

**Comment by amarantha-k** [comment]
The deploy preview looks fine. Waiting to validate landing pages on PR#387 first before merging the PR.
@RichardSmedley @mojavelinux  - Please let me know if any concerns.

**Comment by amarantha-k** [comment]
@mojavelinux Since prod builds are still disabled, we should be okay to merge this PR now, right?

**Comment by mojavelinux** [comment]
That's correct.

**Pull request merged**